drivers/storage/portworx/component/telemetry.go

```diff
@@ -59,16 +83,360 @@ func (t *telemetry) Delete(cluster *corev1.StorageCluster) error {
+		arcusLocation = "rest.cloud-support.purestorage.com"
+	}
+
+	config := fmt.Sprintf(
+		`
+admin:
+  address:
+    socket_address:
+      address: 127.0.0.1
+      port_value: 9901
+
+static_resources:
+  listeners:
+  - name: listener_cloud_support
+    address:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 10000
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          stat_prefix: ingress_http
+          access_log:
+          - name: envoy.access_loggers.stdout
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.access_loggers.stream.v3.StdoutAccessLog
+          http_filters:
+          - name: envoy.filters.http.router
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: local_service
+              domains: ["*"]
+              routes:
+              - match:
+                  prefix: "/"
+                request_headers_to_add:
+                - header:
+                   key: "product-name"
+                   value: "portworx"
+                - header:
+                   key: "appliance-id"
+                   value: "%s"
+                - header:
+                   key: "component-sn"
+                   value: "portworx-metrics-node"
+                - header:
+                   key: "product-version"
+                   value: "%s"
+                route:
+                  host_rewrite_literal: %s
+                  cluster: cluster_cloud_support
+  clusters:
+  - name: cluster_cloud_support
+    type: STRICT_DNS
+    dns_lookup_family: V4_ONLY
+    lb_policy: ROUND_ROBIN
+    load_assignment:
+      cluster_name: cluster_cloud_support
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: %s
+                port_value: 443
+    transport_socket:
+      name: envoy.transport_sockets.tls
+      typed_config:
+        "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
+        common_tls_context:
+          tls_certificates:
+          - certificate_chain:
+              filename: /appliance-cert/cert
+            private_key:
+              filename: /appliance-cert/private_key
+`, cluster.UID, pxutil.GetPortworxVersion(cluster), arcusLocation, arcusLocation)
+
+	data := map[string]string{
+		CollectorProxyConfigFileName: config,
+	}
+
+	return k8sutil.CreateOrUpdateConfigMap(
+		t.k8sClient,
+		&v1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:            CollectorProxyConfigMapName,
+				Namespace:       cluster.Namespace,
+				OwnerReferences: []metav1.OwnerReference{*ownerRef},
+				Annotations:     map[string]string{constants.AnnotationReconcileObject: "true"},
```

Comment:
Actually during test this fails so i added it, the default value is false.

Reply:
I am curious, the test should not fail because of that. There are config maps getting created by other components too. We can check it offline if you want.

Reply:
I think you are right, it was my misunderstanding during test.